manual/program/reviewer_guidelines.rst

```diff
@@ -0,0 +1,167 @@
+
+- Does this person's academic institution, place of employment, or any other affiliations affect how I perceive their work?
+
+
+Things to think about as you prepare to review:
+
+- Bias in pattern-recognition responses — Does one person benefit because they do things “the way we do it around here,” rather than because it is the most innovative, productive, and effective way?
+
+- Pay attention to your projections about the work being evaluated.
+
+- Look for patterns of assessment among different groups. For example, are women in general rated differently than men? 
+
+- Look out for Semantic gender priming: exposure to words more strongly associated with male (e.g., aggressive, competitive) or female (e.g., supportive, nurturing) stereotypes affects subsequent evaluation of male or female targets.[7] 
+
+- Inferences about an individual based on aggregate data or assumptions about a group: making assumptions about individuals based on cultural stereotypes is a type of ecological fallacy. For example, because fewer women lead large center grants, making the assumption that an individual woman is less able to lead a large program; or because overall Black student achievement is lower than for White students, making the assumption that an individual Black student is less prepared.[4]
+
+- Interrupt your bias—refresh yourself after each section as to what rubric/metrics you've established and make sure that you're reviewing a submission accordingly. Be polite. Instead of telling, ask new ways of approach.
+
+- Don't be rude. Always say at least one good thing.
+
+- Be grateful. Point out what you learned and say thanks.
+
+- Stay fresh while reading. Long time reviews might cause some spots to be left uncovered.
+
+- Reviewers have multiple demands on their time, and are frequently under time-constraints to finish reviews, be aware that cognitive biases are efficient, time pressure promotes their influence on decision-making.[4]
+
+
+Other interesting links on the topic about gender bias:
+
+
+`Threats to objectivity in peer review: the case of gender <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552397/&sa=D&ust=1490135409272000&usg=AFQjCNHHxEg8S2lRH-uZYvHLWvQquG3fdg>`_:
+
+`A Linguistic Comparison of Letters of Recommendation for Male and Female Chemistry and Biochemistry Job Applicants <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572075/&sa=D&ust=1490135409273000&usg=AFQjCNG7ODzPnsgxZEaDoxNdNSfjeVskAw>`_:
+
+`Science faculty's subtle gender biases favor male students <https://www.google.com/url?q=http://www.pnas.org/content/109/41/16474.abstract&sa=D&ust=1490135409274000&usg=AFQjCNHeGWyMB7QNHg67j-Blecszk7H9Ig>`_:
+
+`How stereotypes impair women's careers in science <https://www.google.com/url?q=http://www.pnas.org/content/111/12/4403.abstract0&sa=D&ust=1490135409275000&usg=AFQjCNGs6y8FQwel92fbeClDr8_iTHsxUw>`_:
```

Comment:
These all currently end in colons

![image](https://github.com/scipy-conference/scipy-conference/assets/5142394/48fe8cf0-54fd-48a4-a5ba-e9510c09aa10)

Could we drop those and make them a bulleted list instead?
```suggestion
- `Threats to objectivity in peer review: the case of gender <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4552397/&sa=D&ust=1490135409272000&usg=AFQjCNHHxEg8S2lRH-uZYvHLWvQquG3fdg>`_

- `A Linguistic Comparison of Letters of Recommendation for Male and Female Chemistry and Biochemistry Job Applicants <https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2572075/&sa=D&ust=1490135409273000&usg=AFQjCNG7ODzPnsgxZEaDoxNdNSfjeVskAw>`_

- `Science faculty's subtle gender biases favor male students <https://www.google.com/url?q=http://www.pnas.org/content/109/41/16474.abstract&sa=D&ust=1490135409274000&usg=AFQjCNHeGWyMB7QNHg67j-Blecszk7H9Ig>`_

- `How stereotypes impair women's careers in science <https://www.google.com/url?q=http://www.pnas.org/content/111/12/4403.abstract0&sa=D&ust=1490135409275000&usg=AFQjCNGs6y8FQwel92fbeClDr8_iTHsxUw>`_
```